public/manifest.json

```diff
@@ -1,21 +1,11 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "KBase",
```

Comment:
KBase/ui?

Reply:
Updated `name` to be `kbase/ui` and removed `short_name` since `name` is less than 12 characters.